source/module_cell/klist.cpp

```diff
@@ -711,31 +663,46 @@ void K_Vectors::ibz_kpoint(
         ModuleBase::Matrix3 bsymop[48];
         int bnop = 0;
         // search again
-        symm.lattice_type(gb1, gb2, gb3, gb1, gb2, gb3, b_const, b0_const, bbrav, bbrav_name, ucell.atoms, false, nullptr);
+        symm.lattice_type(gb1,
+                          gb2,
+                          gb3,
+                          gb1,
+                          gb2,
+                          gb3,
+                          b_const,
+                          b0_const,
+                          bbrav,
+                          bbrav_name,
+                          ucell.atoms,
+                          false,
+                          nullptr);
```

Comment:
similarly, if you have time, you can change names of these variables, they have really sick names now

Reply:
fine